### PR TITLE
Add email about december virtual tech day

### PR DESCRIPTION
**Pull request by RealOrangeOne**
And asking for interest in possible locations

### CODE REVIEW

File: SR2022/2021-11-22-december-virtual-tech-day.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## December tech day
+
+The first of these tech days will be held on **4th December** in our [Discord](https://studentrobotics.org/docs/team_admin/discord) server. The aim of this day is to help you out by having guaranteed presence from us blueshirts. We'll be around to answer questions about the competition, give kit support, assist with the microgames, and assist with your strategy ideas.
+
+If you are interested in some more direct mentoring on this day, please respond to this email. We can then set up a voice channel for your team in Discord, which can be used both by your team and us mentoring. Note that we may only be in the voice channel if either you, or another parent / guardian is around.
+
+If you have any suggestions of what would be most useful for us to mention or support you on, please also let us know.
+
+## Future tech days
```
Comment (WillB97) [inline SR2022/2021-11-22-december-virtual-tech-day.md:25]:
```suggestion
## Future Tech Days
```

File: SR2022/2021-11-22-december-virtual-tech-day.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+In the new year, we are hoping to be able to run some in-person tech days, allowing some more hands-on mentoring.
```
Comment (WillB97) [inline SR2022/2021-11-22-december-virtual-tech-day.md:27]:
```suggestion
In the new year, we are hoping to be able to run some in-person Tech Days, allowing some more hands-on mentoring.
```

File: SR2022/2021-11-22-december-virtual-tech-day.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+To most efficiently organize and run these, we need to know where would be best to run these events. Please fill in the below form to let us know the locations you would be interested in. We can't confirm dates or schedules yet, but with this information we hope to get some out early January.
+
+[Future tech days form](https://forms.gle/ERruFMEvTWscubM87)
```
Comment (WillB97) [inline SR2022/2021-11-22-december-virtual-tech-day.md:31]:
```suggestion
[Future Tech Days form](https://forms.gle/ERruFMEvTWscubM87)
```

File: SR2022/2021-11-22-december-virtual-tech-day.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+We hope to see you all at a future tech day!
```
Comment (WillB97) [inline SR2022/2021-11-22-december-virtual-tech-day.md:33]:
```suggestion
We hope to see you all at a future Tech Day!
```

**Review by WillB97 — COMMENTED** [review]
Tech Day should be consistently capitalised.

**Comment by antoinepetty** [comment]
Would like to add mentoring info

Dumping suggestion here due to lack of write perms

```
---
to: Student Robotics 2022 teams
subject: Mentoring and December virtual Tech Day and more
---

Hello,

With the competition now underway, we’re planning the ways we can help your
teams build the best robots they can. We can support you through our Tech Days or individual mentoring.

## Mentoring

During the competition year we aim to provide support to teams in many different ways. If you would like one of our volunteers to visit your team meetings on a regular basis, either in person or by video chat, please [sign up for mentoring](https://forms.gle/qqwQkdCfVBSum4w79).

Your mentor will help guide your team towards good solutions for their robot, provide assistance where they might need it as well as help them understand the kit, the rules and the competition as a whole.

If we have volunteers who are able to attend your regular meeting times, we'll let you know.

## Tech Days

Tech Days are opportunities for teams to spend a whole day working on their
robot with lots of help available. They’re also an opportunity to see how other
teams are doing or get more direct help with your robots.

We see a strong correlation between teams to attend Tech Days and performance in the competition.

### December Tech Day

The first of these Tech Days will be held on **4th December** in our [Discord](https://studentrobotics.org/docs/team_admin/discord) server. The aim of this day is to help you out by having guaranteed presence from us blueshirts. We'll be around to answer questions about the competition, give kit support, assist with the microgames, and assist with your strategy ideas.

If you are interested in some more direct mentoring on this day, please respond to this email. We can then set up a voice channel for your team in Discord, which can be used both by your team and us mentoring. Note that we may only be in the voice channel if either you, or another parent / guardian is around.

If you have any suggestions of what would be most useful for us to mention or support you on, please also let us know.

### Future Tech Days

In the new year, we are hoping to be able to run some in-person Tech Days, allowing some more hands-on mentoring.

To most efficiently organize and run these, we need to know where would be best to run these events. Please fill in the below form to let us know the locations you would be interested in. We can't confirm dates or schedules yet, but with this information we hope to get some out early January.

[Future Tech Days form](https://forms.gle/ERruFMEvTWscubM87)

We hope to see you all at a future Tech Day!

```

**Comment by WillB97** [comment]
> If we have volunteers who are able to attend your regular meeting times, we'll let you know.

We may want to reword this line to make it clear that this is after you signup

**Comment by antoinepetty** [comment]
> > If we have volunteers who are able to attend your regular meeting times, we'll let you know.
> 
> We may want to reword this line to make it clear that this is after you signup

New wording is a bit clunky but fixes this

**Review by RealOrangeOne — COMMENTED** [review]
I'm happy with the changes, but someone else should definitely review and approve, too.

**Review by antoinepetty — APPROVED** [review]

**Pull request merged**